andela/mervels-bn-frontend
# #168131522 Configure storybook for component development

#### What does this PR do?
- it configures storybook
#### Description of Task to be completed?
- components should be documented using storybook
#### How should this be manually tested?
- clone the repo
- checkout to `ch-setup-storybook-168131522`
- install all dependencies with `npm i`
- run storybook with `npm run storybook:dev`
- you should see a web page in your browser that has the initial stories
- now on, all components will have stories on storybook
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168131522](https://www.pivotaltracker.com/story/show/168131522)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/39488016/68221636-66994500-fff2-11e9-996f-ef941b142b56.png)


#### Questions:
N/A
